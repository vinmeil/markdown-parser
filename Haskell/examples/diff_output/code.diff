diff --git a/examples/expected_output/code.html b/examples/output/code.html
index ca901bf..d3525cc 100644
--- a/examples/expected_output/code.html
+++ b/examples/output/code.html
@@ -1,17 +1,8 @@
-<!DOCTYPE html>
-<html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <title>Test</title>
-</head>
-
 <body>
     <pre><code class="language-haskell">main :: IO ()
-main = do
+    main = do
     putStrLn "Never gonna give you up"
     putStrLn "Never gonna let you down"
-    putStrLn "Never gonna run around and desert you"</code></pre>
+    putStrLn "Never gonna run around and desert you"
+    </code></pre>
 </body>
-
-</html>
