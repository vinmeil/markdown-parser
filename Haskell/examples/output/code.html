<body>
    <pre><code class="language-haskell">main :: IO ()
    main = do
    putStrLn "Never gonna give you up"
    putStrLn "Never gonna let you down"
    putStrLn "Never gonna run around and desert you"
    </code></pre>
</body>
