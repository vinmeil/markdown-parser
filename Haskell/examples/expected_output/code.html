<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <title>Test</title>
</head>

<body>
    <pre><code class="language-haskell">main :: IO ()
main = do
    putStrLn "Never gonna give you up"
    putStrLn "Never gonna let you down"
    putStrLn "Never gonna run around and desert you"</code></pre>
</body>

</html>
